updatecli/updatecli.d/nginx.yaml

```diff
@@ -30,6 +30,17 @@ sources:
     transformers:
       - trimprefix: "release-"
       - addsuffix: "-alpine"
+  latestStableReleaseMainVersion:
+    name: Get Main Version latest stable version of nginx
+    kind: gittag
+    scmid: nginxGithubMirror
+    spec:
+      versionfilter:
+        kind: regex
+        ## Nginx stable version have the minor digit as an even number
+        pattern: 'release-(\d+)\.(\d*[0|2|4|6|8])\.(\d+)'
+    transformers:
+      - trimprefix: "release-"
```

Comment:
Is there a reason to create a second source of (almost) the same value?

I would have removed the `- addsuffix: "-alpine"` transformer in the existing source, and would have added it (the `addsuffix` transformer) to the condition and the target where it is required instead.

Reply:
you're completely right